serverlist-server/src/main/com/pyratron/pugmatt/bedrockconnect/Server.java

```diff
@@ -4,7 +4,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
 import com.nukkitx.protocol.bedrock.*;
-import com.nukkitx.protocol.bedrock.v422.Bedrock_v422;
+import com.nukkitx.protocol.bedrock.v428.Bedrock_v428;
```

Comment:
Doesn't the source of the package here need updating? 

Also note that [there's a PR](https://github.com/CloudburstMC/Nukkit/pull/1794) for Nukkit (not NukkitX though) that adds support for 1.16.210.

Reply:
I am aware of that PR, but that is based off of the changes done to the Protocol library that this program uses. :)